experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/list.spec.ts

```diff
@@ -653,23 +655,24 @@ describe("List", () => {
 	// TODO: Post-MVP
 
 	describe("implements T[]", () => {
-		describe("Setting [index: number] is disallowed (for MVP)", () => {
-			const subject = createStringList([]);
-
-			assert.throws(() => {
-				(subject as any)[0] = "a";
-			});
+		itWithRoot(
```

Comment:
The pattern of wrapping "it" like this isn't great as it breaks navigate to test, .skip and .only functionality.

I think simply wrapping this in "it" would avoid the above mentioned issues, and also make the test easier to follow for someone not familar with the particular test helpers here.

Reply:
Generally writing code (and utilities) so its easy to use within a regular it test tends to lead to easier to understand libraries and tests than wrapping tests with framework style setup/wrappers like this. It also makes it easier for some tests to differ slightly in approach than others without having to go overcomplicate the wrapper with configuration.

Reply:
After clicking "take me to my test file" and it going to `itWithRoot` for the hundredth time, I think I agree. But why don't we convert all the `itWithRoot`s to `it`s in a separate pass, for consistency.

Reply:
This PR needlessly converts many tests to start using itWithRoot. I don't think we need to stop using itWithRoot in this PR, but porting more tests to use it which seems like something to avoid.